src/app/shared/player/player.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="background-img" [style.background-image]="artworkSafe"></div>
+<div class="background-img" [style.background-image]="feedArtworkSafe"></div>
```

Comment:
So looks like you've added an alternate background image param, in case you don't want to use the same image for foreground/background.

Good feature - I just want to double check that it's (1) settable via a query param, and (2) defaults to the regular `artworkUrl` when not set.